src/bp/ui-studio/src/web/actions/index.ts

```diff
@@ -189,13 +189,14 @@ export const removeFlowNode: (element: any) => void = wrapAction(requestRemoveFl
   }
 })
 
-export const pasteFlowNode: ({ x, y }) => void = wrapAction(requestPasteFlowNode, async (payload, state) => {
+export const pasteFlowNode: ({ x, y }) => void = wrapAction(requestPasteFlowNode, async (payload, state, dispatch) => {
   await updateCurrentFlow(payload, state)
+  dispatch(refreshFlowsLinks())
```

Comment:
Can you explain the reason behind this call?

Reply:
We need the UI can show links between the new nodes, otherwise, the flow needs to be reloaded in order for that to show up